test/clj/milia/api/http_test.clj

```diff
@@ -38,7 +38,7 @@
              (parse-http :method :url)
              => (throws (make-exception-str :no-http-status
                                             "nil"
-                                            :body))
+                                            "nil"))
```

Comment:
Why have we replaced the keyword with a string?

Reply:
Initially, the `make-exception-str` function did not make use of its third argument. In this PR. I have decided, in this PR, to use it to customize the form-params. The other calls to it are now being replaced with "nil".